worldengine/plates.py

```diff
@@ -64,7 +63,7 @@ def world_gen(name, width, height, seed, num_plates=10, ocean_level=1.0,
 
     if verbose:
         start_time = time.time()
-    add_noise_to_elevation(world, random.randint(0, 4096))
+    add_noise_to_elevation(world, numpy.random.randint(0, 4096))
```

Comment:
I believe this is the very first call to the RNG, so currently yes. Some things I left to the "global" RNG since I wasn't sure how to properly use a local one.
Should I change it?